checkov/terraform/checks/module/base_module_check.py

```diff
@@ -11,9 +12,19 @@ def __init__(self, name, id, categories, supported_resources):
         self.supported_resources = supported_resources
         module_registry.register(self)
 
+    def scan_entity_conf(self, conf, entity_type):
+        return self.scan_module_conf(conf, entity_type)
+
     @abstractmethod
-    def scan_resource_conf(self, conf):
+    @multi_signature()
+    def scan_module_conf(self, conf, entity_type):
         raise NotImplementedError()
 
-    def scan_entity_conf(self, conf):
-        return self.scan_resource_conf(conf)
+    @classmethod
+    @scan_module_conf.add_signature(args=["self", "conf"])
+    def _scan_module_conf_self_conf(cls, wrapped):
+        def wrapper(self, conf, entity_type=None):
+            # keep default argument for entity_type so old code, that doesn't set it, will work.
```

Comment:
@angatha Wouldn't the entity_type be always `module` here?

Reply:
Yes